runtime/autoload/health/provider.vim

```diff
@@ -230,10 +229,6 @@ function! s:check_python(version) abort
     if !empty(pythonx_errs)
       call health#report_warn(pythonx_errs)
     endif
-    let old_skip = get(g:, host_skip_var, 0)
-    let g:[host_skip_var] = 1
-    let [python_bin_name, pythonx_errs] = provider#pythonx#Detect(a:version)
-    let g:[host_skip_var] = old_skip
```

Comment:
now `provider#pythonx#Detect()` is not called in this block. Is that what we want?